src/jvm/test.rs

```diff
@@ -1,10 +1,10 @@
-use duchess::Jvm;
+use crate::Jvm;
 
 #[test]
 fn nested_jvm_with() {
     Jvm::with(|_jvm| {
         let err = Jvm::with(|_jvm| Ok(())).expect_err("nested JVMs are illegal");
-        assert!(matches!(err, duchess::Error::NestedUsage));
+        assert!(matches!(err, crate::Error::NestedUsage));
```

Comment:
is this still a useful test since `Jvm::with` is pub_crate now? I guess for our own internal sake we want to avoid nesting it?